src/content/en/fundamentals/performance/rendering/optimize-javascript-execution.md

```diff
@@ -48,7 +48,9 @@ Frameworks or samples may use `setTimeout` or `setInterval` to do visual changes
 
 <img src="images/optimize-javascript-execution/settimeout.jpg" alt="setTimeout causing the browser to miss a frame.">
 
-In fact, jQuery’s default `animate` behavior today is to use `setTimeout`! You can [patch it to use `requestAnimationFrame`](https://github.com/gnarf/jquery-requestAnimationFrame), which is strongly advised.
+In fact, jQuery used to use setTimeout for animate behavior. It was changed to use requestAnimationFrame in version 3.
```

Comment:
eeeeeee thanks for catching that, that was brain farting after 4 hours on video shoot. Fixed now.